lib/list.gd

```diff
@@ -1002,6 +1002,34 @@ DeclareOperation( "PositionsProperty", [ IsList, IsFunction ] );
 DeclareOperation( "PositionBound", [ IsList ] );
 
 
+#############################################################################
+##
+#O  PositionsBound( <list> ) . . . . position of first bound element in a list
```

Comment:
That summary does not seem to fit what the function does -- it returns *all* bound positions, no?